tests/scalers/nats/helper/helper.go

```diff
@@ -0,0 +1,227 @@
+//go:build e2e
+// +build e2e
+
+package helper
```

Comment:
please follow a pattern for file/package naming of a scaler that's used for example here:
https://github.com/kedacore/keda/tree/main/tests/scalers/prometheus